server/helpers/emails/mailer.js

```diff
@@ -12,24 +10,24 @@ module.exports = ({
     //                     but the default is false so just remove this completely
     auth: {
       user,
-      pass,
+      pass
     },
     tls: {
-      rejectUnauthorized: false,
-    },
+      rejectUnauthorized: false
+    }
   });
 
   // send mail with defined transport object
-  if (!(process.env.NODE_ENV === "production" && user.role !== "admin")) {
```

Comment:
Hey @dupreesi I think this is meant to be changed back to the how it used to be right? Having true in there as a condition means it never can get to the transporter.sendMail part of the code.